services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/helpers/Wrappers.groovy

```diff
@@ -0,0 +1,93 @@
+package org.openkilda.functionaltests.helpers
+
+import groovy.util.logging.Slf4j
+import org.junit.runners.model.MultipleFailureException
+
+@Slf4j
+class Wrappers {
+
+    /**
+     * Retries some operation for a given number of times. Retry is considered failed if any throwable is thrown.
+     * Throws MultipleFailureException if closure fails to execute in given amount of tries.
```

Comment:
You could move `Throws MultipleFailureException if closure fails to execute in given amount of tries.` to the javadocs block @throws.